lib/template.js

```diff
@@ -93,6 +93,19 @@ const renderRecursively = (inputJson, template, execOptions = {}) => {
   if (typeof template === 'object' && template !== null) {
     return Object.fromEntries(
       Object.entries(template).flatMap(([key, value]) => {
+        const MY_KEYWORD_ESCAPED = "tempSpreadKeyword\\(\\)";
+        const keywordMatcher = `^\\{\\{\\s*${MY_KEYWORD_ESCAPED}\\s*\\}\\}$`;
+
+        if (key.match(keywordMatcher)) {
```

Comment:
maybe add trim to the key as it can starts/ends with whitespaces as well?

Reply:
The matcher has `\s`s to match whitespaces